hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2522,12 +2522,12 @@
   </property>
 
   <property>
-    <name>ozone.om.layout.version</name>
+    <name>ozone.om.metadata.layout</name>
     <tag>OZONE, OM</tag>
-    <value>V0</value>
+    <value>legacy</value>
     <description>Temporary workaround for OM upgrade and will be replaced once
```

Comment:
Please modify description to something like below. Please feel free to edit, if anything needed.

  ```
  <property>
    <name>ozone.om.metadata.layout</name>
    <tag>OZONE, OM</tag>
    <value>LEGACY</value>
    <description>
      This property is used to define the metadata layout of file system
      paths. If it is configured as PREFIX in combination with
      ozone.om.enable.filesystem.paths to true then this allows to perform
      atomic rename and delete of any directory at any level in the namespace.
      Defaulting to LEGACY. Supported values: LEGACY and PREFIX.
    </description>
  </property>
```

Can we use the value part in capital letters : LEGACY and PREFIX.

In code, the value can be case insensitive and not required to add strict validations.